src/vs/workbench/contrib/notebook/browser/notebook.contribution.ts

```diff
@@ -757,161 +755,160 @@ configurationRegistry.registerConfiguration({
 	title: nls.localize('notebookConfigurationTitle', "Notebook"),
```

Comment:
I added "for VS Code notebooks only" disclaimers to these settings

Reply:
Right but for someone who is using ADS notebooks, there isn't an easy way to filter to just ADS notebook settings. Not a huge deal though since most people are probably searching for a specific setting.

Reply:
I tried changing the label for them so they'd show up as "VS Code Notebooks" under the settings, but that label gets auto generated from the settings key name. So we'd actually have to change all the settings references in code to update that.